lib/v2/screens/authentication/recover/recover_account_found/interactor/mappers/remaining_time_state_mapper.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/screens/authentication/recover/recover_account_found/interactor/viewmodels/current_remaining_time.dart';
+import 'package:seeds/v2/screens/authentication/recover/recover_account_found/interactor/viewmodels/recover_account_found_state.dart';
+
+///Seconds in a day
+const int _daySecond = 60 * 60 * 24;
+
+///Seconds in an hour
+const int _hourSecond = 60 * 60;
+
+///Seconds in a minute
+const int _minuteSecond = 60;
+
+class RemainingTimeStateMapper {
```

Comment:
@RaulUrtecho  if we have multiple places where we need counters, might be useful to make this generic. 
if only 2 places. we can leave like this for now